app/views/questions/show.html.erb

```diff
@@ -85,7 +82,7 @@
                 <%= f.hidden_field :question, value: params[:id] %>
```

Comment:
So I tried this with @question.id instead of params[:id] and it doesn't seem to work, perhaps because the question is not included in the form_for? in that situation it fails in the presenter, probably in the #question method. I think a good sad path test will clear all this up.